tokenomics/etherdai.md

```diff
@@ -0,0 +1,33 @@
+# EtherDai
```

Comment:
I think this page could use a "Motivation" section. Today, we have vaults with liquid staked ETH tokens that generate can DAI. I recall we discussed that the motivation behind EtherDAI was for Maker to keep the staking rewards. Maybe a couple of lines on how exactly this works compared to the current system would help?

Reply:
I could add a stated goals section, similar to the tokenomics pages.